tests/scripts/github_tvmbot.py

```diff
@@ -417,11 +435,16 @@ def author(self) -> str:
         return self.raw["author"]["login"]
 
     def find_failed_ci_jobs(self) -> List[CIJob]:
-        # NEUTRAL is GitHub Action's way of saying cancelled
+        def is_important_job(job):
+            # Ignore GitHub workflows that operate on PRs but don't run tests
+            if job["source"] != "github actions":
+                return True
+            return job["name"].startswith("CI /")
```

Comment:
I might be missing the obvious here, but can you explain how this is tested @driazati ? I've looked through all the fixtures and can't see the string `CI /`?

Reply:
It's the full name of the GitHub actions jobs, e.g. in the signals box on a PR, so its the workflow name + the job name:

<img width="980" alt="image" src="https://user-images.githubusercontent.com/9407960/173432916-c7393d49-ca9b-4643-b587-9132aed59c8b.png">